src/helpers/serviceBusHelper.ts

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line import/no-unresolved
+import { Context } from '@azure/functions';
+
+function getServiceBusOperationId(context: Context): string | undefined {
+  if (context
+    && context.bindingData
+    && context.bindingData.userProperties
+    && context.bindingData.userProperties?.operationId) {
+    return context.bindingData.userProperties?.operationId;
+  }
+  return undefined;
```

Comment:
Any reason this whole thing can't just be chained?
```suggestion
  return context?.bindingData?.userProperties?.operationId;
```

Reply:
Changed to use chained calls. I'm stilling wrapping to always return undefined, instead of null for ?.operationId & ?.parentId